examples/run_glue.py

```diff
@@ -227,6 +232,11 @@ def train(args, train_dataset, model, tokenizer):
 
                     for key, value in logs.items():
                         tb_writer.add_scalar(key, value, global_step)
+
+                    if args.log_on_wandb and _WANDB_AVAILABLE:
+                        epoch = global_step / (len(train_dataloader) // args.gradient_accumulation_steps)
+                        wandb.log({**logs, "epoch": epoch})
```

Comment:
We could add `step=global_step` here for good measure.

Reply:
The issue I've been having with `step` is that it does not commit when using it so I need to add a `wandb.log({})` to update my last step.